docs/advanced_examples/FullyConnectedNeuralNetwork.ipynb

```diff
@@ -62,16 +62,13 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": 16,
    "metadata": {},
    "outputs": [],
    "source": [
     "params = {\n",
     "    \"module__n_layers\": 3,\n",
-    "    \"module__n_w_bits\": 3,\n",
-    "    \"module__n_a_bits\": 4,\n",
-    "    \"module__n_accum_bits\": 9,\n",
-    "    \"module__activation_function\": nn.Sigmoid,\n",
+    "    \"module__activation_function\": nn.ReLU,\n",
```

Comment:
Why was this needed? Changing the architecture might have a positive impact on the accuracy which is not really what we want in this PR.

Reply:
I believe this is because the power of two scaling feature only works with ReLU

Reply:
Got it. But I don't see anything that prevents the use of sigmoid or other non linear function. What would happen then?